model-mesh-triton-adapter/README.md

```diff
@@ -6,26 +6,33 @@ Written in Golang.
 
 ## How to
 
-1.  Pull Triton Serving Docker Image
+1. Clone the repository
```

Comment:
Mind adding another space before "Clone" to pass the lint check?

Reply:
Done